src/config/functions.ts

```diff
@@ -1,12 +1,14 @@
 import * as Bluebird from 'bluebird';
 import * as _ from 'lodash';
+import { fs } from 'mz';
```

Comment:
We could use `fs.promises` as we're on node >= 10

Reply:
We still have a node 6 device :(